### ISSUE TITLE
Location of "Open in new window" at top/bottom

### ISSUE THREAD

**Issue by lonix1** [takkaO/VSCode-OpenWindowTabContext#8]
Firstly: thanks for this extension. It's great! :+1:

---
The "Open in new window" is at the top of the context menu.

I am sure some people like that.

But please keep in mind that it's customary for vscode items to be at the top, and "extra" items (added by extensions) to be at the bottom. So sometimes it's confusing to find it because I'm expecting to find it at the bottom. :-)

Maybe a setting can be used: "top" or "bottom" ?

**Comment by takkaO** [comment]
**Thank you for using this extension.**

From version 1.5.0, the display position of the context menu can be adjusted.  
You can adjust it to your liking from the Extensions settings.

The default settings are shown at the top to avoid confusing previous users (and also for my preference).

**Comment by lonix1** [comment]
I just upgraded, and it works nicely - thanks! :smile: 

May I ask, why must that setting be in "user settings", rather than "workspace settings" (most extensions are in workspace settings: ["scope": "window"](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema))... is there a technical reason for that?

(It's nice to be able to commit the settings in `.vscode/settings.json` to source control together with the project.)

...I'm just curious, not complaining! :wink:

**Comment by takkaO** [comment]
There is no specific meaning for scope.
Since I added this setting based on IronFractal's code, I didn't really think about the significance of the scope item.

I read the explanation in the link, but I can't image the difference between the scopes. 
Is there any reference to know the difference in more detail?

**Comment by lonix1** [comment]
Yes, [here's the docs](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations) for user vs workspace settings...

- A setting in "user settings" applies to all vscode instances. See link above for file paths. For example, in linux it's `$HOME/.config/Code/User/settings.json`. This is `"scope": "application"`. This extension is like that.
- A setting in "workspace settings" applies to a particular workspace. It's saved in `.vscode/settings.json`. This is better because you can add the config to git together with your code. Much better.

Most extensions save in "user" **AND** "workspace" (`"scope": "window"`), so you can specify the setting wherever you want. [See here](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema) and scroll down to the section named "scope".

You can test this yourself. You can edit your workspace settings `.vscode/settings.json` and add the settings. It will give you an error saying that you must move that setting to the user settings `$HOME/.config/Code/User/settings.json`.

TLDR; it's better to leave scope at the default (`"scope": "window"`), so users can set it however they want.

(If you decide to look into this, I'm happy to be your beta tester.)

**Comment by takkaO** [comment]
Thank you very much.
After some debug, we plan to use ``scope:window`` in the next version.

**Comment by lonix1** [comment]
No problem. I'm looking forward to the next version.
And again, thanks for this excellent extension.

**Comment by takkaO** [comment]
I have released a version 1.6.0 that incorporates this issue, although it was delayed due to the implementation of another feature.

**Comment by lonix1** [comment]
It works nicely - thanks. And I like the new auto-close feature.

We can close this issue now?
